src/performance-analysis-spec/specific-hardware-and-software.md

```diff
@@ -38,7 +37,8 @@ avoiding triggering of stalls stemming from PCC-relative accesses depending on
 new PCC bounds as described in [Morello microarchitectural
 limitations](../performance-methodology/morello-microarchitectural-limitations.md).
 This workaround should especially recover non-essential overhead associated
-with jump tables and calls into short functions.
+with jump tables and calls into short functions &mdash; a change that is now
+also used in baseline pure-capability code generation.
```

Comment:
... so it should be described in the purecap ABI section, otherwise you have to read the benchmark ABI section to discover that, despite the PCC issue being explicitly called out for jump tables in the purecap ABI section, it is not in fact the ABI that's used even there.